app.js

```diff
@@ -6,3 +6,52 @@
   // get user input
   // use user input to update state 
   // update DOM to reflect the new state
+
+//SIMPLE ARRAY OF STRINGS - GUITARS
+
+import { guitars } from './array-guitars.js';
+import { renderGuitars } from './render-guitars.js';
+
+const list1Div = document.querySelector('#list1');
+
+for (let eachGuitar of guitars) {
+    const p = renderGuitars(eachGuitar);
+    list1Div.append(p);
+}
+
+
+//ARRAY OF OBJECTS - AMPS
+
+import { amps } from './array-amps.js';
+import { renderAmps } from './render-amps.js';
```

Comment:
Great work on all of this, Clayton!  Just a note:  as kind of 'best practice', I would do all of your importing at the top of the file.  Sometimes certain templates or projects will have a linting rule setup that only allows imports at the top of a file, so it is a good habit to get into.